_docs_integrate/migration-from-v4-to-v5.md

```diff
@@ -72,46 +73,58 @@ The aspects to be taken into account when migrating to version 5, which were bri
 
 ### DIDs as Addresses
 
-The address format changed from `<3-character realm><32- or 33-character base58-string>` to `did:e:<backbone-hostname>:dids:<22-character lowercase hex string>`.
+Each [Identity]({% link _docs_integrate/data-model-overview.md %}#identity) within enmeshed has a unique `address` for identification. The format of this `address` has changed from `<3-character realm><32- or 33-character Base58-encoded string>` to `did:e:<hostname of Backbone>:dids:<22-character lowercase hexadecimal string>`. Accordingly, the `realm` property of the Identity was removed as well. Furthermore, the two error codes `error.runtime.MultiAccount.WrongRealm` and `error.transport.identity.realmLength` have been removed.
```

Comment:
```suggestion
Each [Identity]({% link _docs_integrate/data-model-overview.md %}#identity) within enmeshed has a unique `address` for identification. The format of this `address` has changed from `<3-character realm><32-or33-character-base58-encoded string>` to `did:e:<backbone-hostname>:dids:<22-character-lowercase hexadecimal-string>`. Accordingly, the `realm` property of the Identity was removed as well. Furthermore, the two error codes `error.runtime.MultiAccount.WrongRealm` and `error.transport.identity.realmLength` have been removed.
```

Reply:
I'm not yet happy with the formatting of these placeholders :D I'll see what I can do about it.

Reply:
Aaaand... I couldn't do anything about it :D When we use the placeholder notation `<...>` we usually separate words by spaces and not by hyphens. Also, we use the same capitalization rules as in the regular paragraphs (for example, capitalize `'Backbone'`). The page about enmeshed addresses is an exception where the placeholders are currently used differently than we usually do. For reasons of consistency, however, I have undone my changes as discussed with you above. I also looked up proper names such as `'hexadecimal string'` and checked whether hyphens should naturally be used there, which was not the case in the example.

Reply:
however, base58 should not be capitalized ;)

Reply:
Alrighty, you're the captain! ⛵